### ISSUE TITLE
Label doesn't show in iPhone

### ISSUE THREAD

**Issue by xrax** [ChimpStudios/AmCharts#13]
This Api is wonderfull!

I have a problem to show label in iPhone (iPad works fine) 

AmSerialChart * chart = [[AmSerialChart alloc] init];
...
AmLegend* legend = [[AmLegend alloc] init];
[chart setLegend:legend];
...

I see the javascriptRepresentation of final chart, it's fine in both (iPhone and iPad). I try resize the AmMobileChartView too. Anyway, I couldn't run any legend in iPhone :c. 

Thanks!

**Comment by AndrewSchenk** [comment]
If it is working on iPad but not iPhone it is probably a matter of view size.  If the view is too small, some of the chart elements might be rendering out of bounds.

**Comment by xrax** [comment]
Yes, how I said... I tried resize the view, and I tried set enable scroll in this view... but doesn't works :s

**Comment by AndrewSchenk** [comment]
If it is generating valid javascript representation and works in iPad it is something wrong with your iPhone layout.  End of story.  There's very little I can do about that.

**Comment by xrax** [comment]
It doesn't an iPhone layout, I have an auto layout for everyone (iPod, iPhone, iPad) anyway I still search a solution. Thanks.

**Comment by AndrewSchenk** [comment]
http://www.amcharts.com/tutorials/fitting-pie-charts-small-containers/

**Comment by xrax** [comment]
Hi again... I tried put a small container, still doesn't works yet... 

I update the issue with more information. Thanks.
